### PR TITLE
Set DotNetCoreCLIV2 task's default verbosity input as Minimal

### PR DESCRIPTION
**Pull request by ivanduplenskikh**
**Task name**: DotNetCoreCLIV2

**Description**: **verbosity** input is set as **Minimal** ([it is default value](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-restore#options) for dotnet restore command)
Related [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/12298).
PR is recreated since the source repo was deleted.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** #19848

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/task.json [hunk 1]
```diff
@@ -262,7 +262,7 @@
             "name": "verbosityRestore",
             "type": "pickList",
             "label": "Verbosity",
-            "defaultValue": "Detailed",
+            "defaultValue": "Minimal",
```
Comment (cremor) [inline Tasks/DotNetCoreCLIV2/task.json:265]: It was previously decided that the default value should be `Normal`. See https://github.com/microsoft/azure-pipelines-tasks/issues/12297#issuecomment-582506056

**Review by vmapetr — APPROVED** [review]

**Comment by cremor** [comment]
Yes. And not only the green YAML code, but the text "Default value: Detailed." in the text below too.

**Comment by ivanduplenskikh** [comment]
It looks like we [don't need to update](https://github.com/MicrosoftDocs/azure-devops-docs/pull/12595#issuecomment-1239455003) the documentation

**Comment by cremor** [comment]
@ivanduplenskikh The documentation page was updated 2 days ago (after this PR was merged) but it still shows the old default values.

**Comment by steved0x** [comment]
> @ivanduplenskikh The documentation page was updated 2 days ago (after this PR was merged) but it still shows the old default values.
> 
> **edit:** Maybe this is because the updated task is not yet actually used in pipelines?

There is a PR pending with the latest task updates in the documentation that includes the new default value and it should be live in a day or two. I run the tooling every 2 weeks and it picks up the changes from my test organization. I have been on vacation so there was a little bit of a break in running the tool.

**Pull request merged**